CONTRIBUTING.md

```
@@ -153,29 +134,49 @@ Do NOT use Svelte 4 stores (`writable`, `readable`, `derived` from `svelte/store
+
+Rackula uses the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) rather than a Contributor Licence Agreement (CLA). The DCO is a lightweight, one-line certification that you have the right to submit your contribution under the project's MIT licence. There is no paperwork and no copyright assignment: you keep the copyright to your work.
+
+To certify a contribution, sign off your commits with the `-s` flag:
+
+```bash
+git commit -s -m "feat: add rail snapping"
+```
+
+This appends a trailer to the commit message:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)